### DISCUSSION TITLE
Does useTranslate function in vue support multiple namespaces?

### DISCUSSION THREAD

**Discussion by AlvinWang627** [tolgee/tolgee-platform#2696] [Q&A] [answered]
[useTranslate](https://docs.tolgee.io/js-sdk/integrations/vue/api#composable-usetranslate)
When I use the useTranslate function with multiple namespaces like this:
```
const { t } = useTranslate(['order', 'product', 'general']);
```
The function call t.value('addFile') correctly fetches the translation from the `order namespace`. 
However, t.value('addProduct') does not work as expected for the `product namespace`.

If I switch the product namespace to be first in the array, like this:
```
const { t } = useTranslate(['product', 'order', 'general']);
```
product namespace is work, order namespace is not work.

Can someone help me ? where is the failure?

**Comment by stepan662** [comment, marked as answer]
Hey, this is expected behavior, we are using the first namespace listed. If you use more than one the others needs to be referenced by `t('addProduct', { ns: 'product' })`

For some reason it's not mentioned in the docs, however it's an intended behavior.

I'll fix the docs.

**Reply by AlvinWang627** [reply]
ok I understand, thank you for your reply.
